### PR TITLE
Edit delete chat message#139

### PR DESCRIPTION
**Pull request by kamyh**

### CODE REVIEW

**Comment by kamyh** [comment]
corrections phpcs

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,5 @@ bootstrap/cache/*
 
 #Vagrant
 .vagrant
+/.idea/
```
Comment (ice-blaze) [inline .gitignore]: This should be in another issue. But now we leave it here.

File: resources/views/chats/index.blade.php [hunk 2]
```diff
@@ -9,6 +9,16 @@
     <div class="row">
         @forelse($chats as $chat)
             <li class="list-group-item">
+                @if($chat->isEditable($chat->sender_id))
+                <div class="delete-message">
```
Comment (ice-blaze) [inline resources/views/chats/index.blade.php]: Identation

File: .gitignore (hunk 1 above)
Comment (ice-blaze) [inline .gitignore]: This should be in another issue. But now we leave it here.

File: app/Http/Controllers/ChatUserController.php [hunk 3]
```diff
@@ -39,4 +40,33 @@ public function store(Request $request, $reciever_id)
 
         return back();
     }
+
+    public function edit(ChatUser $chat)
+    {
+        if (Auth::User()->id != $chat->sender_id) {
+            return redirect('/404');
```
Comment (ice-blaze) [inline app/Http/Controllers/ChatUserController.php]: Isn't tested and is 404 really the best place to redirect user ? I would redirect to back with flash error, or juste back or home.

File: tests/ChatUserTest.php [hunk 4]
```diff
@@ -56,4 +56,43 @@ public function testChatUserMethodes()
...
+        $this->actingAs($alice);
+        $this->visit($bob->path());
+        $this->click('Chat with this talent');
+        $this->type($message, 'content');
+        $this->press('send');
+        $this->seePageIs($bob->path().'/chat');
+        $this->see($message);
+
+        $this->click('delete_message');
+        $this->visit($bob->path());
```
Comment (ice-blaze) [inline tests/ChatUserTest.php]: wrong path checked ``$bob->path().'/chat'` for example or use URL

**Pull request merged**